src/utils/re_frame.cljs

```diff
@@ -88,3 +88,9 @@
 (def dispatch-sync re-frame/dispatch-sync)
 
 (def reg-event-fx re-frame/reg-event-fx)
+
+(defn call-continuation
+  [continuation & args]
+  (cond
+    (vector? continuation) (dispatch (into continuation args))
+    (fn? continuation)     (apply continuation args)))
```

Comment:
Here's a new helper function for handling callbacks and vectors for continuations inside re-frame events/effects.

Reply:
I'm not sure what to name this function since we pass two kinds of _continuations_:
1. a **callback function** (useful for situations where we need to call a function and cannot pass an event vector)
2. an **event vector** (useful for forwarding results into a chain of event handlers)

Reply:
I like the proposal @seanstrom.

On the other hand, most of the time, I personally prefer to create more events for the `on-success`/`on-error` handlers and then something like `call-continuation` is not needed. This would come at the expense of a bit more verbosity, but doing that eliminates the black box anonymous functions (simplifies testing) and increases visibility because dispatched events are logged (also helps re-frisk).

Naming this is hard. I'm biased towards a more concrete name, like `call-or-dispatch`, but that's not great either, but `call-continuation` is okay.

I'd just consider adding a good docstring and a simple example.

Also, the function should probably throw if whatever is passed is not a vector or function instead of silently doing nothing because that would most certainly be a bug in prod, but since we don't currently have a good way of displaying general errors to users, we can at least log at the error level?

Reply:
I don't know full context of code but still it's confusing for me, or if you can add more description about how it works. also it will be good to have some more eyes here maybe @ilmotta can have some suggestion.

Reply:
Thanks for the feedback 🙏 
I'll add some docs / examples for this code 📖